gradio/blocks.py

```diff
@@ -671,21 +671,19 @@ def load(
         For reverse compatibility reasons, this is both a class method and an instance
         method, the two of which, confusingly, do two completely different things.
 
-        Class method: loads a demo from a Hugging Face Spaces repo and creates it locally
-        Parameters:
-            name (str): the name of the model (e.g. "gpt2"), can include the `src` as prefix (e.g. "models/gpt2")
-            src (str | None): the source of the model: `models` or `spaces` (or empty if source is provided as a prefix in `name`)
-            api_key (str | None): optional api key for use with Hugging Face Hub
-            alias (str | None): optional string used as the name of the loaded model instead of the default name
-            type (str): the type of the Blocks, either a standard `blocks` or `column`
-        Returns: Blocks instance
-
-        Instance method: adds an event for when the demo loads in the browser.
+
+        Class method: loads a demo from a Hugging Face Spaces repo and creates it locally and returns a block instance.
+
+
+        Instance method: adds an event for when the demo loads in the browser and returns None.
         Parameters:
-            fn: Callable function
-            inputs: input list
-            outputs: output list
-        Returns: None
+            name: Class Method - the name of the model (e.g. "gpt2"), can include the `src` as prefix (e.g. "models/gpt2")
```

Comment:
I think adding the load method to the docs page will increase it's visibility and help with the original problem this issue tracks, which is that users don't know they can load Blocks demos.

Only tricky thing is that this is a class and instance method, and the current documentation building code does not naturally display two separate parameters tables. So I decided to combine them.

![image](https://user-images.githubusercontent.com/41651716/179293535-8a8b596a-c02e-4b52-ae28-6d7191f204c6.png)

Reply:
👍